src/crimson/os/seastore/cache.cc

```diff
@@ -1561,7 +1561,13 @@ void Cache::complete_commit(
       is_inline = true;
       i->set_paddr(final_block_start.add_relative(i->get_paddr()));
     }
-    assert(i->get_last_committed_crc() == i->calc_crc32c());
+#ifndef NDEBUG
+    if (!i->get_paddr().is_absolute() || epm.get_checksum_needed(i->get_paddr())) {
```

Comment:
The extent `i` is about to be added to the shared `Cache` index below which should almost be deterministic.

I guess another possible case that the paddr isn't absolute is because `i` may be ROOT (not really sure).

Let's try remove `is_absolute()` here and see how the UTs complain.